br/pkg/metautil/metafile.go

```diff
@@ -553,7 +545,16 @@ func (writer *MetaWriter) FinishWriteMetas(ctx context.Context, op AppendOp) err
 	return nil
 }
 
-func (writer *MetaWriter) flushBackupMeta(ctx context.Context) error {
+//
+func (writer *MetaWriter) FiniallyFlushBackupMeta(ctx context.Context) error {
```

Comment:
I perfer to rename this method name to `FlushBackupMeta` and add a method comment

Reply:
Oh yeah, I do it now!